### DISCUSSION TITLE
Crackling noises

### DISCUSSION THREAD

**Discussion by larspalo** [GrandOrgue/grandorgue#213] [General]
Originally posted 2018-07-14 15:42:34.004000 by ziggi55 in GrandOrgue discussion forum on SourceForge<br><br>Hi, I am very new to GO and this forum. I pretty much just finished a 3-manual VPO and while using a couple of GO samples (mainly Pitea), I hear a faint crackling noise when, and only when, the music is playing, and only in the left channel. The sound is very much like fire kindling burning and crackling. The sound does not happen in Hauptwerk or any other audio application on this PC. The PC is a 16GB, 3+ GHz Win 7 machine, with the NVIDIA High Definition Audio board. 

The problem happens also when the Win system is empty of any other apps (no virus scan running, low CPU usage). I tweaked the bit resolution and the sample rate in GO, no change. Also, the sound is basically the same with any level of polyphony. I recorded a short sample with only one rank-one melody, and the sound is there. 

I found some hits searching this discussion, but nothing describing my problem exactly.  Can you comment or help?

Thank you very much in advance, Zdenek Gavlas.

**Comment by larspalo** [comment]
Originally posted 2018-07-16 18:43:35.071000 by [Lars Palo](https://github.com/larspalo) in GrandOrgue discussion forum on SourceForge<br><br>Hi! You&#039;ll have to give a little more details on the problem since many different things could be causing the issue in question. So, does this left channel crackling occur on every sampleset you load, on every sample (on all stops and on each key) or is it more specific? Does it change in any way? Is it repeating after some time duration on a held note? Are you using sound settings in GO that are as close to the recording settings of the samples in the sampleset as is possible on your setup?

**Comment by larspalo** [comment]
Originally posted 2018-07-17 13:42:09.704000 by ziggi55 in GrandOrgue discussion forum on SourceForge<br><br>Hi, Lars. Thanks for taking my question. Up till now, I thought the crackling happened on every sample set, but just now, I went back to your GO Demo sample set, and the sound is NOT there! It does, however, appear on any other set (Burea, Pitea, Cracow). The sound is not related to any rank, or note. It is universal for all of them. It starts a second after the first note is played, and ends 1-1,5 sec after the last note is released. To me as a programmer, it sounds like it is related to some kind of buffer cleanup.  

Now, going to your last question, I don&#039;t recall adjusting each sample set&#039;s parameters for any new I downloaded, and that may be the problem. I am probably still on the default setting (also, since the Demo is playing cleanly). I did change a couple of times the sample rate and the bit resolution, just to see if there is a difference. There was none. Now I am on 24bit, 96000 sample rate.

Thanks,
Zdenek

**Reply by larspalo** [reply]
Originally posted 2018-07-18 10:57:01.128000 by [Lars Palo](https://github.com/larspalo) in GrandOrgue discussion forum on SourceForge<br><br>Test changing the samplerate to 44100 for the problematic samplesets and
see if anything changes.

Den tis 17 juli 2018 15:42Zdenek Gavlas &lt;ziggi55@users.sourceforge.net&gt;
skrev:

&gt; Hi, Lars. Thanks for taking my question. Up till now, I thought the
&gt; crackling happened on every sample set, but just now, I went back to your
&gt; GO Demo sample set, and the sound is NOT there! It does, however, appear on
&gt; any other set (Burea, Pitea, Cracow). The sound is not related to any rank,
&gt; or note. It is universal for all of them. It starts a second after the
&gt; first note is played, and ends 1-1,5 sec after the last note is released.
&gt; To me as a programmer, it sounds like it is related to some kind of buffer
&gt; cleanup.
&gt;
&gt; Now, going to your last question, I don&#039;t recall adjusting each sample
&gt; set&#039;s parameters for any new I downloaded, and that may be the problem. I
&gt; am probably still on the default setting (also, since the Demo is playing
&gt; cleanly). I did change a couple of times the sample rate and the bit
&gt; resolution, just to see if there is a difference. There was none. Now I am
&gt; on 24bit, 96000 sample rate.
&gt;
&gt; Thanks,
&gt; Zdenek
&gt; ------------------------------
&gt;
&gt; Crackling noises
&gt; &lt;https://sourceforge.net/p/ourorgan/discussion/962124/thread/bcfa3248/?limit=25#1506&gt;
&gt; ------------------------------
&gt;
&gt; Sent from sourceforge.net because you indicated interest in
&gt; https://sourceforge.net/p/ourorgan/discussion/962124/
&gt;
&gt; To unsubscribe from further messages, please visit
&gt; https://sourceforge.net/auth/subscriptions/
&gt;
<br><br>Attachments in original post:<br>https://sourceforge.net/p/ourorgan/discussion/962124/thread/bcfa3248/1506/4828/attachment/alternate<br>

**Reply by larspalo** [reply]
Originally posted 2018-07-19 12:57:08.434000 by ziggi55 in GrandOrgue discussion forum on SourceForge<br><br>Hi, Lars. Your suggestion did fix the problem. Thanks for that. But I noticed that with the lower sampling rate, a fast one-note staccato yields a random spike on NOTE-ON, sometimes almost like a pop. If this is an inevitable consequence of lowering the sampling rate, I think I can live with it. Unless there is a good soultion for that, as well.
Best Regards,
Zdenek

**Reply by larspalo** [reply]
Originally posted 2018-07-19 16:58:16.536000 by [Lars Palo](https://github.com/larspalo) in GrandOrgue discussion forum on SourceForge<br><br>The next things I would experiment with is the driver selected for your
audio device and both the samples per buffer setting and the desired
latency of your audio device. The default settings might not be optimal on
your setup. Unfortunately I cannot say what will work best for you, you&#039;ll
have to test that for yourself. Also remember that many of the free
samplesets might have different issues or even faults in samples or
metadata.

Den tor 19 juli 2018 14:57Zdenek Gavlas &lt;ziggi55@users.sourceforge.net&gt;
skrev:

&gt; Hi, Lars. Your suggestion did fix the problem. Thanks for that. But I
&gt; noticed that with the lower sampling rate, a fast one-note staccato yields
&gt; a random spike on NOTE-ON, sometimes almost like a pop. If this is an
&gt; inevitable consequence of lowering the sampling rate, I think I can live
&gt; with it. Unless there is a good soultion for that, as well.
&gt; Best Regards,
&gt; Zdenek
&gt; ------------------------------
&gt;
&gt; Crackling noises
&gt; &lt;https://sourceforge.net/p/ourorgan/discussion/962124/thread/bcfa3248/?limit=25#1506/4828/11c0&gt;
&gt; ------------------------------
&gt;
&gt; Sent from sourceforge.net because you indicated interest in
&gt; https://sourceforge.net/p/ourorgan/discussion/962124/
&gt;
&gt; To unsubscribe from further messages, please visit
&gt; https://sourceforge.net/auth/subscriptions/
&gt;
<br><br>Attachments in original post:<br>https://sourceforge.net/p/ourorgan/discussion/962124/thread/bcfa3248/1506/4828/11c0/be7b/attachment/alternate<br>

**Comment by larspalo** [comment]
Originally posted 2018-07-19 21:54:52.739000 by ziggi55 in GrandOrgue discussion forum on SourceForge<br><br>Thank you. Will try all that.
Zdenek

**Comment by schiavo** [comment]
I resolved this by lowering the volume in Grand Orgue. I could then raise the volume on the computer without crackling noises.
